examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>Maybe not mention this if you are not going to explore where they are occurring...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<blockquote>(Left as an exercise for the reader)</blockquote><p>?</p><p> </p><p>I'd like to keep this line because, while I'm skipping the diagnosing step and going right to the marginalized model, these divergences are still indicating a pathological geometry. I want to point that out.</p>

Reply:
Fine by me then :)